.deps/EXCLUDED/dev.md

```diff
@@ -0,0 +1,33 @@
+This file contains a manual contribution to .deps/dev.md and it's needed because eclipse/dash-licenses does not deal with work-with CQs (more see https://github.com/eclipse/dash-licenses/issues/13)
```

Comment:
Welcome back, automated dependencies analizying!

It would be great if we expose it from README.md, like:

### Dependencies IP

To make sure all the dependencies satisfy Eclipse Intellectual Property, this repo uses https://github.com/che-incubator/dash-licenses which is a wrapper for https://github.com/eclipse/dash-licenses.

So, check .deps/EXCLUDED/prod.md for dependencies use package and .deps/dev.md for ones we use at build time.

Reply:
I have fixed it.